src/gui/screen_menu_tune.cpp

```diff
@@ -161,9 +164,11 @@ int screen_menu_tune_event(screen_t *screen, window_t *window,
         case MI_BABYSTEP:
             z_offs = psmd->items[MI_BABYSTEP].item.wi_spin_fl.value;
             break;
-        case MI_MESSAGES:
+            // je potreba mit ? - stara se o to menu_screen_event
```

Comment:
please keep comments in EN
And processing MI_MESSAGES is not necessary in this menu